lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -4494,20 +4494,28 @@ def model_apply_prm_baseline_window_to_wall_ratio(model, climate_zone, wwr_build
         # of each space conditioning category (res, nonres, semi-heated)
         # separately.  Include space multipliers.
         bat.store('nr_wall_m2', 0.001) # Avoids divide by zero errors later
+        bat.store('nr_fene_only_wall_m2', 0.001)
+        bat.store('nr_plenum_wall_m2', 0.001)
         bat.store('nr_wind_m2', 0)
         bat.store('res_wall_m2', 0.001)
+        bat.store('res_fene_only_wall_m2', 0.001)
         bat.store('res_wind_m2', 0)
+        bat.store('res_plenum_wall_m2', 0.001)
         bat.store('sh_wall_m2', 0.001)
+        bat.store('sh_fene_only_wall_m2', 0.001)
         bat.store('sh_wind_m2', 0)
+        bat.store('sh_plenum_wall_m2', 0.001)
         bat.store('total_wall_m2', 0.001)
-        bat.store('total_subsurface_m2', 0.0)
+        bat.store('total_plenum_m2', 0.001)
       else
         bat = bat_win_wall_info[std_spc_type]
       end
 
       # Loop through all surfaces in this space
       wall_area_m2 = 0
       wind_area_m2 = 0
+      wall_only_area_m2 = 0
+      num_wall_no_fene = 0
```

Comment:
Is this unused?

Reply:
Yes, the `num_wall_no_fene` is not used.
It was part of implementation on handling windows + door surface area but I quickly move to another implementation approach.
Must forget to delete it. Good catch, thanks.